MSAL/src/MSALPublicClientApplication.m

```diff
@@ -377,7 +377,7 @@ + (void)logOperation:(NSString *)operation
     if (result)
     {
         NSString *hashedAT = [result.accessToken msalShortSHA256Hex];
-        LOG_INFO(ctx, @"%@ returning with at: %@ scopes:%@ expiration:%@", operation, hashedAT, result.scopes, result.expiresOn);
+        LOG_INFO(ctx, @"%@ returning with at: %@ scopes:%@ expiration:%@", operation, _PII(hashedAT), result.scopes, result.expiresOn);
```

Comment:
nit: I don't think this macro itself is easy to understand from it's name. Maybe we should consider renaming it?